src/app/sections/header/header.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="headerBar">
+  <img class="logoLetters" src="../../../assets/Logo/Logotipo-Plasentir-Letters-White.png" alt="Plasentir">
+
+  <div class="menuAction">
+    <h2 class="menuTittle">Sobre nosotros</h2> |
+    <span class="icoShoppingCart material-symbols-outlined">shopping_cart</span> |
+    <div class="menuButton">
+      <h2 class="menuTittle">Menu</h2>
+      <span class="material-symbols-outlined">menu</span>
+    </div>
+  </div>
+
+</div>
```

Comment:
It could be better to use this material [component](https://material.angular.io/components/toolbar/overview).